manage

```diff
@@ -339,22 +262,25 @@ main() {
     fi
 
     case "$cmd" in
-        --getenv) var="$1"; echo "${!var}";;
-        --help) help;;
-        --*)
-            help
-            err_msg "unknown option $cmd"
+    --getenv)
+        var="$1"
+        echo "${!var}"
+        ;;
+    --help) help ;;
+    --*)
+        help
+        err_msg "unknown option $cmd"
+        return 42
+        ;;
+    *)
+        _type="$(type -t "$cmd")"
+        if [ "$_type" != 'function' ]; then
```

Comment:
I think these modifications here are only whitespaces .. may related to your editor settings.  Some editors interpret one tab as 4 spaces (a tab is by definition 8 spaces). To avoid tab problem, we don't use tab for indentation (except in Makefiles, where a tab is needed).

For shell files we use 4 spaces for one indentation:

https://github.com/searxng/searxng/blob/bc06b1aece43c50fa0dae7e6d90389814bebbc91/.editorconfig#L5-L7

Can you please check your editor settings.